patches/react-native-webview+10.3.2.patch

```diff
@@ -93,6 +93,8 @@ index ab869cf..2aa7a9e 100644
 +          }
 +        };
 +        task.execute();
++      } else {
++        request.cancel();
```

Comment:
It's similar to https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/2777

@newmanw @jangaraj which one is correct?

Reply:
@newmanw for me it doesn't make sense to cancel request `request.cancel();`.

It should proceed `super.onReceivedClientCertRequest(view, request);`. Server will handle that request was made without client certificate (for example in my case my IDP offers standard username/password authentication instead of mutual TLS; in other cases server return). IMHO that should be default option. Real world examples: https://www.programcreek.com/java-api-examples/?api=android.webkit.ClientCertRequest

Reply:
@diegolmello @jacotec I apologize I missed the other PR.

With the current Android implementation either is correct, as android.webkit.WebViewClient.onReceivedClientCertRequest()  will call request.cancel() if its not overridden, which is what used to happen.

request.cancel() only cancels the auth challenge request and is what the client should do in the case where there is not a client certificate to pass to the server.

https://android.googlesource.com/platform/frameworks/base/+/a8352f4/core/java/android/webkit/WebViewClient.java#L313

`
public void onReceivedClientCertRequest(WebView view, ClientCertRequest request) {
request.cancel();
}
`  

I am glad this came up as I went back and forth on the "right" fix.  I am ok with calling super and going with what Android provides as the default.  It will only matter if Android changes its implementation, which is currently request.cancel().

Reply:
@newmanw can you change your PR to call super then as #2777 does?
I remember @jangaraj was having some issues there and that's why it's still a WIP.

Reply:
Thank you for the explanation. @diegolmello so it's up to you what do you want to implement. Both options (`request.cancel()`, `super()`) resolve my problem.

Reply:
@jangaraj agreed, both PRs match now and call super class implementation in the case of no client cert.  Wouldn't hurt my feelings if you close this PR, thanks for finding this @jangari.

@diegolmello I appreciate you looking into this.  Currently all our android users are unable to authenticate as our authentication provider displays an alternate way to login if a client cert is not present.  However that page will never load without this fix.  Possible to bump this in priority?

Reply:
@newmanw Yes, we've been working on fixes only and the plan is to release a patch next week.
I'm merging this now.

@jangaraj I'm adding you as co-author of this PR.

Thanks for your PRs.